packages/demo/src/pages/Main/Main.tsx

```diff
@@ -0,0 +1,101 @@
+import { ToastContainer } from "react-toastify";
+
+import { Bandada } from "../../components/Bandada/Bandada";
+import { Connect } from "../../components/Connect/Connect";
+import { ConnectedIdentity } from "../../components/ConnectedIdentity/ConnectedIdentity";
+import { DisplayProof } from "../../components/DisplayProof/DisplayProof";
+import { GetCommitment } from "../../components/GetComitment/GetComitment";
+import { GetMetadata } from "../../components/GetMetadata/GetMetadata";
+import { RateLimitingNullifier } from "../../components/RateLimitingNullifier/RateLimitingNullifier";
+import { Semaphore } from "../../components/Semaphore/Semaphore";
+import { VerifiableCredentials } from "../../components/VerifiableCredentials/VerifiableCredentials";
```

Comment:
Better to rename files to `index.tsx` so we can import them like this:

```suggestion
import { Bandada } from "../../components/Bandada";
import { Connect } from "../../components/Connect";
import { ConnectedIdentity } from "../../components/ConnectedIdentity";
import { DisplayProof } from "../../components/DisplayProof";
import { GetCommitment } from "../../components/GetComitment";
import { GetMetadata } from "../../components/GetMetadata";
import { RateLimitingNullifier } from "../../components/RateLimitingNullifier";
import { Semaphore } from "../../components/Semaphore";
import { VerifiableCredentials } from "../../components/VerifiableCredentials";
```